assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -88,24 +91,41 @@ const EditorWizardModal = () => {
 		} );
 	};
 
-	// TODO Implement different flows depending on post type 👇.
-	const steps = [ CourseDetailsStep, UpgradeStep, CoursePatternsStep ];
-
-	const onWizardCompletion = ( data ) => {
-		// TODO Implement actions when wizard is completed.
-
-		// eslint-disable-next-line no-console
-		console.log(
-			`Wizard completed with data: ${ JSON.stringify( data ) }`
-		);
+	// Choose steps by post type.
+	const stepsByPostType = {
+		course: [ CourseDetailsStep, UpgradeStep, CoursePatternsStep ],
+		lesson: [ LessonDetailsStep, LessonPatternsStep ],
+	};
+	const postType = select( 'core/editor' )?.getCurrentPostType();
+	const steps = stepsByPostType[ postType ];
```

Comment:
Nice solution to get the steps of the wizard by post type.

Maybe it would be worth to add a comment to backlink [to this file](https://github.com/Automattic/sensei/blob/086323830efcda4c10a2add2928b80a7a3cfdb87/includes/admin/class-sensei-editor-wizard.php#L83) (and maybe add a link there too) so we can avoid a situation where the editor wizard is included on a post type that is not supported. 

WDYT?

Reply:
Thanks for your comment! Instead of this I have updated the code to just ignore a non-supported post type.
In case we add a new post type and we want to show some wizard it should be pretty obvious how to find this and change it.